proto/observer/params.proto

```diff
@@ -19,11 +20,21 @@ message CoreParams {
   string zeta_token_contract_address = 8;
   string connector_contract_address = 9;
   string erc20_custody_contract_address = 10;
-  int64 chain_id = 11;
   int64 outbound_tx_schedule_interval = 12;
   int64 outbound_tx_schedule_lookahead = 13;
+  string ballot_threshold = 14 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  string min_observer_delegation = 15 [
```

Comment:
Bringing @kingpinXD here. 
Do you think it makes sense to have a per-chain min_observer_delegation? 
If no i'd suggest moving it out of `CoreParams`

Reply:
I think we can create an issue if it's a change that makes sense.

Reply:
makes sense to remove the per chain , min_delgation amount as we are moving to a unified observer set now

Reply:
@kingpinXD can you create an issue for it?